specs/experimental/op-contracts-manager.md

```diff
@@ -20,146 +20,247 @@ of governance approved [contract releases] can be found on the
+chain that complies with the [standard configuration]. It has the following interface:
+
+```solidity
+/// @notice Deploys a new OP Chain
+/// @param _input DeployInput containing chain specific config information.
+/// @return DeployOutput containing the new addresses.
+function deploy(DeployInput calldata _input) external returns (DeployOutput memory)
 ```
 
 The `l2ChainId` has the following restrictions:
 
 - It must not be equal to 0.
 - It must not be equal to the chain ID of the chain the OP Contracts Manager is
-deployed on.
+  deployed on.
 - It must not be equal to a chain ID that is already present in the
-[ethereum-lists/chains] repository. This is not enforced onchain, but may matter
-for future versions of OP Contracts Manager that handle upgrades.
+  [ethereum-lists/chains] repository. This is not enforced onchain, but may matter
+  for future versions of OP Contracts Manager that handle upgrades.
 
 On success, the following event is emitted:
 
 ```solidity
-event Deployed(uint256 indexed l2ChainId, SystemConfig indexed systemConfig);
+event Deployed(uint256 indexed outputVersion, uint256 indexed l2ChainId, address indexed deployer, bytes deployOutput);
 ```
 
 This method reverts on failure. This occurs when:
 
 - The input `l2ChainId` does not comply with the restrictions above.
 - The resulting configuration is not compliant with the [standard configuration].
 
-### Getter Methods
+### Implementation
 
-The following interface defines the available getter methods:
-
-```solidity
-/// @notice The logic address and initializer selector for an implementation contract.
-struct Implementation {
-  address logic; // Address containing the deployed logic contract.
-  bytes4 initializer; // Function selector for the initializer.
-}
-
-/// @notice Returns the latest approved release of the OP Stack contracts.
-/// @notice Release strings follow semver and are named with the
-/// format `op-contracts/vX.Y.Z`.
-function latestRelease() external view returns (string memory);
-
-/// @notice Maps a release version to a contract name to its implementation data.
-function implementation(
-  string memory release,
-  string memory contractName
-) external view returns (Implementation memory);
-
-/// @notice Maps an L2 Chain ID to the SystemConfig address for that chain.
-/// @notice All contracts for a chain can be found from its SystemConfig.
-function systemConfig(uint256 chainId) external view returns (SystemConfig);
-```
-
-## Implementation
-
-### Batch Inbox Address
+#### Batch Inbox Address
 
 The chain's [Batch Inbox] address is computed at deploy time using the recommend approach defined
 in the [standard configuration]. This improves UX by removing an input, and ensures uniqueness of
 the batch inbox addresses.
 
-### Contract Deployments
+#### Contract Deployments
 
 All contracts deployed by the OP Contracts Manager are deployed with CREATE2, with a
```

Comment:
This would read better if it said 

```
All contracts deployed by the OP Contracts Manager are deployed with CREATE2.

For singletons the following salt is used:

...

For `Proxy` contracts, the following salt is used:

...
```

This removes ambiguity around former/latter, especially for esl speakers

Reply:
good rewrite tyvm: 6de833da8ce2